etc/X11/Xsession.d/65snappy

```diff
@@ -0,0 +1,7 @@
+# This file is sourced by Xsession(5), not executed.
+# Add additionnal the additonal snappy desktop path
+
+XDG_DATA_DIRS=$XDG_DATA_DIRS:/var/lib/snappy/desktop
```

Comment:
Should this say $.../var/lib/snappy/desktop/applications or is "applications" magically hardcoded by some XDG spec elsewhere?